src/Containers/KitchenCard/KitchenCard.tsx

```diff
@@ -6,23 +6,21 @@ import { Status, StatusColors } from '../Status';
 import { Paragraph as P } from '../../Text';
 import { flex, media } from '../../Utils/Mixins';
 import { KitchenCardItems } from './KitchenCardItems';
+import { Item } from './constants';
 
 const CUSTOMER_FIRST_NAME = /^([\w-]+)/g;
 const UNDERSCORE_FORMAT = '_';
 const LABEL_FORMAT = -4;
 
-interface Item {
-    name: string;
-}
-
-export interface KitchenProps {
+export interface KitchenCardProps {
     customer: {
         name: string;
     };
     _id: string;
     items: Item[];
     orderType: string;
-    status: StatusColors;
+    statusColor: StatusColors;
```

Comment:
Having these two variables seperate could lead to showing the wrong color for a different status, wondering if there is a better way to make this error proof or if CDD requires this

Reply:
Because the way the status component is using themeing to do "red" "green" "orange" an enum will need to be created to match the appropriate status color to the status being passed in.

I implemented this in the storybook. I think unless you error in typing the enum it will be error proof? I might be missing something.

![StatusColorFilter](https://user-images.githubusercontent.com/29719870/94716099-3a21a200-0303-11eb-8e8d-f97238679cdd.png)

Reply:
It's not ideal but it works i guess